index.js

```diff
@@ -7,7 +7,16 @@ var Q = require('q');
 var cordovaLib = require('cordova-lib').cordova;
 var cordova = cordovaLib.raw;
 
-module.exports = function (rm) {
+module.exports = function (rm, version) {
```

Comment:
I would rather see this implemented with an options object instead of those 2 parameters.

```js
module.exports = function (opts) {
    opts = opts || {};

    // and so on
}
```

I'm ok with `rm.version`, not sure about `reAdd` though. Any other suggestions for this?